inc/classes/class-srm-post-type.php

```diff
@@ -448,17 +453,17 @@ public function action_register_post_types() {
 
 		$redirect_capability = 'srm_manage_redirects';
 
-        $roles = array( 'administrator' );
+		$roles = array( 'administrator' );
 
-        foreach ( $roles as $role ) {
+		foreach ( $roles as $role ) {
 			$role = get_role( $role );
 
 			if ( empty( $role ) || $role->has_cap( $redirect_capability ) ) {
 				continue;
 			}
 
-            $role->add_cap( $redirect_capability );
-        }
+            	$role->add_cap( $redirect_capability );
```

Comment:
Might as well replace the spaces with tabs.

Reply:
Done